Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -7427,15 +7429,21 @@ - (BOOL)isKeyboardScrollingAnimationRunning
 }
 
 - (void)_executeEditCommand:(NSString *)commandName
+{
+    [self _executeEditCommand:commandName notifyDelegate:WebKit::NotifyInputDelegate::Yes];
+}
+
+- (void)_executeEditCommand:(NSString *)commandName notifyDelegate:(WebKit::NotifyInputDelegate)notifyDelegate
 {
     _autocorrectionContextNeedsUpdate = YES;
     // FIXME: Editing commands are not considered by WebKit as user initiated even if they are the result
     // of keydown or keyup. We need to query the keyboard to determine if this was called from the keyboard
     // or not to know whether to tell WebKit to treat this command as user initiated or not.
-    [self _internalBeginSelectionChange];
-    RetainPtr<WKContentView> view = self;
-    _page->executeEditCommand(commandName, { }, [view] {
-        [view _internalEndSelectionChange];
+    if (notifyDelegate == WebKit::NotifyInputDelegate::Yes)
+        [self _internalBeginSelectionChange];
+    _page->executeEditCommand(commandName, { }, [view = retainPtr(self), notifyDelegate] {
```

Comment:
just curious; What's the change in this `view` capture about?

Reply:
Ah, just decided to make this a bit more concise while I was here.